public/app/features/explore/TraceView/createSpanLink.tsx

```diff
@@ -362,3 +361,27 @@ function getTimeRangeFromSpan(
     },
   };
 }
+
+// Interpolates span attributes into trace to metric query, or returns default query
+function buildMetricsQuery(query: TraceToMetricQuery, tags: Array<KeyValue<string>> = [], span: TraceSpan): string {
+  if (!query.query) {
+    return `histogram_quantile(0.5, sum(rate(tempo_spanmetrics_latency_bucket{operation="${span.operationName}"}[5m])) by (le))`;
```

Comment:
I'm curious why this is the query that's used if we don't have a query..
1. why this query
2. under what circumstances would we not have a query (the query.query type is not optional)

Reply:
1. It's a reasonable default if users are generating span metrics and shows the average latency for that particular span. 
2. It'd be pretty uncommon, but it'll get used if the query is blank

Reply:
I'll make the query.query optional to indicate that.